enrollment-server-onboarding/src/main/java/com/wultra/app/onboardingserver/statemachine/StateMachineConfig.java

```diff
@@ -329,14 +334,21 @@ private void configurePresenceCheckTransitions(StateMachineTransitionConfigurer<
                 .and()
                 .withExternal()
                 .source(OnboardingState.PRESENCE_CHECK_IN_PROGRESS)
-                .event(OnboardingEvent.EVENT_NEXT_STATE)
+                .event(OnboardingEvent.PRESENCE_CHECK_SUBMITTED)
+                .action(moveToPresenceCheckVerificationPendingAction)
+                .target(OnboardingState.PRESENCE_CHECK_VERIFICATION_PENDING)
+
+                .and()
+                .withExternal()
+                .source(OnboardingState.PRESENCE_CHECK_VERIFICATION_PENDING)
```

Comment:
@kober32 You may expect `PRESENCE_CHECK / VERIFICATION_PENDING` at the status endpoint.